roles/nova-data/tasks/monitoring.yml

```diff
@@ -13,6 +13,7 @@
   sensu_metrics_check: name=check-cinder-sessions plugin=check-cinder-sessions.py
                        interval=10
   notify: restart sensu-client
+  when: "'rbd_volumes' not in cinder.enabled_backends"
```

Comment:
This may not work as it's looking for a variable out of cinder role. Maybe try:

`when: "'rdb_volumes' not in cinder.enabled_backends|(default([])"`